lib/block/block.js

```diff
@@ -62,27 +51,35 @@ Block._from = function _from(arg) {
 };
 
 /**
- * @param {String|Object} - A JSON string or object
+ * @param {String} - A JSON string
  * @returns {Object} - An object representing block data
  * @private
  */
 Block._fromJSON = function _fromJSON(data) {
```

Comment:
Reading the code I realized many classes don't have a `fromObject` method. I removed the Object option on those because I added a `fromObject` method. IMO we should have separate `fromJSON`/`fromObject` methods in each class. `fromJSON` most commonly will be equivalent to `Foo.fromObject(JSON.parse(input))`